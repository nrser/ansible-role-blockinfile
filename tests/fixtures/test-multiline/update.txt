111 222 333

##############################################################################
# BEGIN MULTILINE SECTION
#
xxx yyy zzz
XXX YYY ZZZ
#
# END MULTILINE SECTION
##############################################################################

!!! @@@ ###
