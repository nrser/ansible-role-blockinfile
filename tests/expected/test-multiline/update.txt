111 222 333

##############################################################################
# BEGIN MULTILINE SECTION
#
aaa bbb ccc
AAA BBB CCC
#
# END MULTILINE SECTION
##############################################################################

!!! @@@ ###
